src/ig/IGExporter.ts

```diff
@@ -976,14 +1031,15 @@ export class IGExporter {
               } else {
                 this.ig.definition.resource.push(newResource);
               }
-              const inputPath = path.join(dirPath, file);
-              const outputPath = path.join(
-                igPath,
-                'input',
-                pathEnd,
-                `${resourceJSON.resourceType}-${resourceJSON.id}${path.extname(file)}`
-              );
-              this.copyAsIs(inputPath, outputPath);
+
+              if (this.shouldCopyFiles) {
+                const inputPath = path.join(dirPath, file);
+                const outputFileName = this.isLegacyIgPubContext
+                  ? `${resourceJSON.resourceType}-${resourceJSON.id}${path.extname(file)}`
+                  : file;
```

Comment:
I'm not sure I understand here. Under what conditions is this not a `LegacyIgPubContext`?

Reply:
Hm this is a good point. I had changed that line before we had gotten to the current configuration for output and hadn't thought to go back to it. I think that the check for `isLegacyIgPubContext` is wrong, but I actually think that it should have supported this name change in any legacy configuration, not just a legacy IG Publisher configuration. So I think getting rid of the ternary operator here and only supporting the rename is correct. And the surrounding `if (this.shouldCopyFiles)` will support the current configuration. This means some of the tests will need to change as well, so I'll make those changes  now.

Reply:
I updated the test to specifically check that no files are copied, which I believe is the expected behavior. Any file that should be copied is from a legacy tank configuration and therefore should be renamed, so I got rid of the ternary operator on this line.